### PR TITLE
Changes a bunch of New()s to Initialize()s

### PR DESCRIPTION
**Pull request by Cyberboss**
Maints, why aren't you stopping dis :(

(This isn't even half of them)

### CODE REVIEW

**Comment by tralezab** [comment]
😂 Who 😂 did 😂 this 😂

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -408,8 +408,8 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
 	layer = HIGH_LANDMARK_LAYER
 
 
-/obj/effect/landmark/event_spawn/New()
-	..()
+/obj/effect/landmark/event_spawn/Initialize(mapload)
+	. = ..()
 	GLOB.generic_event_spawns += src
```
Comment (optimumtact) [inline code/game/objects/effects/landmarks.dm:413]: have you verified nothing is trying to access this list before atom subsystem inits?

File: code/_onclick/hud/new_player.dm [hunk 2]
```diff
@@ -15,6 +16,8 @@
 	layer = LOBBY_BUTTON_LAYER
 	screen_loc = "TOP,CENTER"
 
+/atom/movable/screen/lobby/proc/SlowInit()
```
Comment (Mothblocks) [inline code/_onclick/hud/new_player.dm:19]: Can you doc-comment this?

Also add a `return` here just so it doesn't look out of place as being an empty proc.

File: code/_onclick/hud/screen_objects.dm [hunk 3]
```diff
@@ -673,7 +673,11 @@
 	plane = SPLASHSCREEN_PLANE
 	var/client/holder
 
-/atom/movable/screen/splash/New(client/C, visible, use_previous_title) //TODO: Make this use INITIALIZE_IMMEDIATE, except its not easy
+//TODO: Make this use INITIALIZE_IMMEDIATE, except its not easy
```
Comment (Mothblocks) [inline code/_onclick/hud/screen_objects.dm:676]: Looks like you did, you can remove this TODO now

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/atmosmachinery.dm [hunk 4]
```diff
@@ -70,15 +70,15 @@
 		if(HAS_TRAIT(L, TRAIT_VENTCRAWLER_NUDE) || HAS_TRAIT(L, TRAIT_VENTCRAWLER_ALWAYS))
 			. += span_notice("Alt-click to crawl through it.")
 
-/obj/machinery/atmospherics/New(loc, process = TRUE, setdir, init_dir = ALL_CARDINALS)
+/obj/machinery/atmospherics/Initialize(mapload, process = TRUE, setdir, init_dir = ALL_CARDINALS)
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/machinery/atmosmachinery.dm:73]: I feel like there was some reason for this, but I don't quite remember it. Fuck

Comment (Cyberboss) [reply code/modules/atmospherics/machinery/atmosmachinery.dm:73]: On second thought, it'd be insane to do this without doing all the atmos machines at once

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 5]
```diff
@@ -265,7 +265,7 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	appearance_flags = TILE_BOUND
 	vis_flags = NONE
 
-/obj/effect/overlay/gas/New(state, alph)
+/obj/effect/overlay/gas/Initialize(mapload, state, alph)
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/gasmixtures/gas_types.dm:268]: This also makes me just a mite nervous, but I'm pretty sure any issues would fall out pretty fast so it should be fine

**Comment by Mothblocks** [comment]
im feelin lucky

**Pull request merged**